src/lib.rs

```diff
@@ -306,6 +306,9 @@ const MAX_ACK_RANGES: usize = 68;
 // The highest possible stream ID allowed.
 const MAX_STREAM_ID: u64 = 1 << 60;
 
+/// The default max_datagram_size in the congestion control.
+pub const MAX_DATAGRAM_SIZE: usize = 1200;
```

Comment:
This shouldn't be `pub`, as it shouldn't be exported in the quiche public API I think. It's probably also a good idea to rename this to `MAX_SEND_UDP_PAYLOAD_SIZE` to avoid confusion with DATAGRAM as well (it's fine to leave the `recovery` code as is though, since that's what the variable is called in the recovery draft).
```suggestion
const MAX_SEND_UDP_PAYLOAD_SIZE: usize = 1200;
```